src/js/components/Select/Select.js

```diff
@@ -92,28 +96,30 @@ class Select extends Component {
           onClose={this.onClose}
           a11yTitle={`${a11yTitle}${typeof value === 'string' ? `, ${value}` : ''}`}
           dropContent={<SelectContainer {...this.props} onChange={onSelectChange} />}
+          fill={true}
         >
           <Box
-            aria-hidden={true}
             align='center'
             border={!plain ? 'all' : undefined}
             direction='row'
             justify='between'
           >
-            {selectValue || (
-              <TextInput
-                style={{ cursor: 'pointer' }}
-                ref={(ref) => { this.inputRef = ref; }}
-                {...rest}
-                tabIndex='-1'
-                type='text'
-                placeholder={placeholder}
-                plain={true}
-                size={size}
-                readOnly={true}
-                value={textValue}
-              />
-            )}
+            <Box direction='row' flex={true}>
+              {selectValue || (
+                <TextInput
+                  style={{ cursor: 'pointer' }}
```

Comment:
Why do we have this here instead of in the styling? I realize this isn't part of this change. So, you can ignore if you want to.